arch/risc-v/src/mpfs/Kconfig

```diff
@@ -773,6 +773,35 @@ config MPFS_CANFD_DATA_BITRATE1
 	default 4000000
 	depends on MPFS_CANFD1
 
+comment "MSS CAN 2.0 Options"
+
+config MPFS_HAVE_MSS_CAN
+	bool "MSS CAN 2.0 peripheral"
```

Comment:
You can drop the description in this line, no need to select this specifically. You can just select this if configuring MPFS_MSS_CAN0 or MPFS_MSS_CAN1.